demo/apisix/tmail-apisix-plugin-runner/src/main/java/com/linagora/apisix/plugin/AppConfiguration.java

```diff
@@ -16,11 +18,12 @@
 public class AppConfiguration {
     private final Logger logger = LoggerFactory.getLogger(AppConfiguration.class);
 
-    public static StringRedisTemplate redisTemplate(String host, int port) {
+    public static StringRedisTemplate redisTemplate(String host, int port, Optional<String> password) {
         var redisConfiguration = new RedisStandaloneConfiguration();
```

Comment:
Can this `RedisStandaloneConfiguration` work with the Master/Slave topology we are using?

Reply:
NO?
"Standalone" here is client-side. 
And...
![image](https://user-images.githubusercontent.com/81145350/236429820-989bea9a-9bcb-4ef8-bf25-332ff654864f.png)

Reply:
@vttranlina I think @quantranhong1999 makes sense.

I don't say remove the standalone, it's perfect for the demo for example, but we need to manage master/slave mode as well, if we want to use it on preprod/prod

Reply:
> "Standalone" here is client-side.

We have only one endpoint of Redis on preprod/prod for james, right? 
I think we must be care of another topo when we have >=2 endpoint

Reply:
I remember it being not that simple... Pretty sure we thought like this first for Redis in James as well and we had to do the differentiation at the end :)

Reply:
Actually you write only on master but should be able to read from the slaves I believe?

Reply:
let me check the similarities in James' code